src/physics.cpp

```diff
@@ -626,16 +624,15 @@ void absorption(Particle& p, int i_nuclide)
 {
   if (settings::survival_biasing) {
     // Determine weight absorbed in survival biasing
-    p.wgt_absorb() = p.wgt() * p.neutron_xs(i_nuclide).absorption /
-                     p.neutron_xs(i_nuclide).total;
+    double wgt_absorb = p.wgt() * p.neutron_xs(i_nuclide).absorption /
```

Comment:
Maybe use `const double`, since it's slightly more clear and the compiler can sometimes leverage const-ness.